backend/wallets/utilities.py

```diff
@@ -0,0 +1,46 @@
+from typing import Type
+from rest_framework.request import Request
+from django.db.models import Model
+from django.core.exceptions import ObjectDoesNotExist, PermissionDenied
+from django.http import Http404
+
+class HandlersChecks:
+
+    @staticmethod
+    def check_ownership(request: Request, object_class: Type[Model], instance_id: int) -> None:
```

Comment:
Those can probably be just permission methos to be used as permission_classes as I mentioned in another comment with the reference on how to do it

Reply:
Right, I moved it to permission_classes. I used it in my first project structure, with aggregated viewssets, but I modified it while making structure changes.